### PR TITLE
[TEVA-1248] Update Organisation Details Page and Notification

### PR DESCRIPTION
**Pull request by josephhull676**
## Jira ticket URL

https://dfedigital.atlassian.net/browse/TEVA-1248

### Trust Details

Screenshot 2020-09-09 at 16 38 22

### School Details

Screenshot 2020-09-09 at 16 39 23

### Notification Change

Screenshot 2020-09-09 at 16 40 03

### CODE REVIEW

**Review by alexbowen — APPROVED** [review]

File: app/views/hiring_staff/organisations/schools/edit.html.haml [hunk 1]
```diff
@@ -12,11 +12,12 @@
...
 
       %h2.govuk-heading-l
-        = t('.title')
+        = t('.title', organisation: organisation_type_basic(@organisation))
 
       = f.govuk_text_area :description,
-        label: { text: t('helpers.fieldset.organisation_form.description'), size: 's' },
-        hint_text: t('helpers.hint.organisation_form.description'),
+        label: { text: t('helpers.fieldset.organisation_form.description', organisation: organisation_type_basic(@organisation)), size: 's' },
+        hint_text: t('helpers.hint.organisation_form.description', organisation: organisation_type_basic(@organisation)),
```
Comment (david-mears-dfe) [inline app/views/hiring_staff/organisations/schools/edit.html.haml:19]: Feature/system tests are time-expensive, so I'm OK with the logic of 'trust' vs 'school' going untested.

**Review by david-mears-dfe — APPROVED** [review]
I have one quibble, but it looks good and I've approved.

File: config/locales/en.yml [hunk 2]
```diff
@@ -235,7 +235,7 @@ en:
           Give a brief summary of the schools where the jobseeker would be teaching and any relevant information about
           the trust
       organisation_form:
-        description: Jobseekers will see this description of the school in the job listing
+        description: Jobseekers will see this description of the %{organisation} in the job listing
```
[the remark this answers is not in the document]
Comment (josephhull676) [reply config/locales/en.yml:238]: Sure, I'll change these now.

File: app/helpers/vacancies_helper.rb [hunk 3]
```diff
@@ -106,9 +106,9 @@ def vacancy_about_school_label_organisation(vacancy)
 
   def vacancy_about_school_hint_text(vacancy)
     return I18n.t('helpers.hint.job_summary_form.about_schools') if vacancy.organisations.many?
-    return I18n.t('helpers.hint.job_summary_form.about_organisation', organisation: 'Trust') if
+    return I18n.t('helpers.hint.job_summary_form.about_organisation', organisation_type: 'Trust') if
```
Comment (cpjmcquillan) [inline app/helpers/vacancies_helper.rb:109]: You can get rid of this line

File: app/helpers/vacancies_helper.rb [hunk 4]
```diff
@@ -106,9 +106,9 @@ def vacancy_about_school_label_organisation(vacancy)
...
       vacancy.parent_organisation.is_a?(SchoolGroup)
-    I18n.t('helpers.hint.job_summary_form.about_organisation', organisation: 'School')
+    I18n.t('helpers.hint.job_summary_form.about_organisation', organisation_type: 'School')
```
Comment (cpjmcquillan) [inline app/helpers/vacancies_helper.rb:111]: And just do this
```ruby
I18n.t('helpers.hint.job_summary_form.about_organisation', organisation_type: organisation_type_basic(vacancy.parent_organisation))
```

**Pull request merged**